### DISCUSSION TITLE
How to use with Google calendar

### DISCUSSION THREAD

**Discussion by SETHAN52** [bitfireAT/icsx5#455] [Q&A] [answered]
Android user here. I got a calendar subscribe to via URL and now having troubles using Google calendar to access it.

**Comment by ArnyminerZ** [comment]
Hi, what's your issue? Do you already have the URL, or you are trying to export it from Google Calendar?

**Reply by SETHAN52** [reply]
Once I subscribed to url inside icsx5, how do I get it to show up in Google calendar?

**Reply by MachineMind-Tech** [reply]
Following

**Reply by sunkup** [reply, marked as answer]
It shows up in the sidebar under "Calender Subscriptions". You can enable and disable it there.

![Screenshot from 2024-11-18 11-42-38](https://github.com/user-attachments/assets/b1beeac3-41ad-498a-85b6-9ba93093c05e)

**Reply by SuperKenVery** [reply]
Even after I tick it, the events still did not show up. Did anyone meet this?
